doc/fundamentals/append_overwrite.rst

```diff
@@ -1,571 +1,364 @@
-   ``--split lines --target-file-bytes 100``, you’ll see the input file
-   split into about 10 files or so, each of which will have 5 or so
-   lines. Each split-file’s size will hover above the target value of
-   100 bytes, not going below 100 bytes until the last split-file, which
-   may be less than 100 bytes.
-
--  ``--target-file-datums`` will attempt to fill each split-file with
-   the number of datums you specify. Going back to that same
-   line-delimited 50-line file above, if you use
-   ``--split lines --target-file-datums 2``, you’ll see the file split
-   into 50 split-files, each of which will have 2 lines.
-
--  Specifying both flags, ``--target-file-datums`` and
-   ``--target-file-bytes``, will result in each split-file containing just
-   enough data to satisfy whichever constraint is hit first. Pachyderm
-   will split the file and then fill the first target split-file with
-   line-based records until it hits the record limit. If it passes the
-   target byte number with just one record, it will move on to the next
-   split-file. If it hits the target datum number after adding another
-   line, it will move on to the next split-file. Using the example
-   above, if the flags supplied to ``put file`` are
-   ``--split lines --target-file-datums 2 --target-file-bytes 100``, it
-   will have the same result as ``--target-file-datums 2``, since that’s
-   the most compact constraint, and file sizes will hover around 40
-   bytes.
-
-What split data looks like in a Pachyderm repository
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Going back to our 50-line file example, let’s say that file is named
-“my-data.txt”. We’ll create a repo named “line-data” and load
-my-data.txt into Pachyderm with the following commands:
-
-::
-   
-   $ pachctl create repo line-data
-   $ pachctl put file line-data@master -f my-data.txt --split line
+line-delimited text files, JavaScript Object Notation (JSON) files,
+Pachyderm includes the `--split`, `--target-file-bytes`, and
+`--target-file-datums` flags.
 
-After ``put file`` is complete, list the files in the repo.
+**Note:** In this section, a chunk of data is called a *split-file*.
 
-::
-   
-   $ pachctl list file line-data@master
-   COMMIT                           NAME         TYPE COMMITTED          SIZE
-   8cce4de3571f46459cbe4d7fe222a466 /my-data.txt dir  About a minute ago 1.071KiB
+| Flag              | Description                                           |
+| ----------------- | ----------------------------------------------------- |
+| `--split`         | Divides a file into chunks based on a *record*. In <br>line-delimited files, it is a line. In JSON files, <br>it is an object. The `--split` flag takes one argument, such as `line`, <br> `json`, or `sql`. This argument defines how to split the file into <br>chunks. For example, if you use `--split line`, Pachyderm divides <br>your file on new line boundaries and not in the middle of a line. |
```

Comment:
This repeates itself a few times. 

>  Divides a file into chunks based on a *record*. In <br>line-delimited files, it is a line. In JSON files, <br>it is an object. The `--split` flag takes one argument, such as `line`, <br> `json`, or `sql`. This argument defines how to split the file into <br>chunks. For example, if you use `--split line`, Pachyderm divides <br>your file on new line boundaries and not in the middle of a line.

 Divides a file into chunks based on a *record* such as newlines in a line-delimited file or by json object for json files. The `--split` flag takes one argument, `line`, <br> `json`, csv, or `sql`. For example, `--split line` ensures Pachyderm will only break up a file on newline boundaries and not in the middle of a line.